src/decorators/arbAggregatorActions.integration.test.ts

```diff
@@ -0,0 +1,60 @@
+import { it, expect, describe } from 'vitest';
+import { createPublicClient, http } from 'viem';
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts';
+
+import { nitroTestnodeL2 } from '../chains';
+import { arbAggregatorActions } from './arbAggregatorActions';
+import { getNitroTestnodePrivateKeyAccounts } from '../testHelpers';
+
+const testnodeAccounts = getNitroTestnodePrivateKeyAccounts();
+const l2RollupOwner = testnodeAccounts.l2RollupOwner;
+const randomAccount = privateKeyToAccount(generatePrivateKey());
+
+const nitroTestnodeL2Client = createPublicClient({
+  chain: nitroTestnodeL2,
+  transport: http(),
+}).extend(arbAggregatorActions);
+
+describe('ArgAggregator decorator tests', () => {
+  it('successfully fetches the batch posters and the fee collectors', async () => {
+    const batchPosters = await nitroTestnodeL2Client.arbAggregatorReadContract({
+      functionName: 'getBatchPosters',
+    });
+
+    expect(batchPosters).toHaveLength(2);
+    expect(batchPosters[0]).toEqual('0xA4b000000000000000000073657175656e636572');
+
+    const batchPosterFeeCollector = await nitroTestnodeL2Client.arbAggregatorReadContract({
+      functionName: 'getFeeCollector',
+      args: [batchPosters[0] as `0x${string}`],
```

Comment:
This should not be needed now, return type should be infered (same below on L45 and 56)